src/Console/ConfigurationResolver.php

```diff
@@ -350,6 +350,10 @@ private function parseRules()
         $rules = array();
 
         foreach (array_map('trim', explode(',', $this->options['rules'])) as $rule) {
+            if ('' === $rule) {
+                throw new InvalidConfigurationException('Empty rule name is not allowed.');
+            }
+
             if ('-' === $rule[0]) {
```

Comment:
We might want to be more careful here, since someone could have any number of `-` in front.